docker-compose.yml

```diff
@@ -4,6 +4,18 @@
 
 version: '2'
 services:
+  lando-ui.test:
+    image: jwilder/nginx-proxy
+    container_name: lando-ui.test
+    volumes:
+      - /var/run/docker.sock:/tmp/docker.sock:ro
```

Comment:
Tested, it is necessary. It seems to be how the nginx box connects to the other docker containers to begin proxying.